src/FilterListContainer/FilterList.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ChangeEventHandler, Component } from 'react';
+
+import './FilterList.css';
+
+interface FilterProps {
+  title: string;
+  name: string[];
+  changeEvent: ChangeEventHandler<HTMLSelectElement>;
+}
+
+class FilterList extends Component<FilterProps> {
+  render() {
+    const { title, name, changeEvent } = this.props;
+    const option = name.map((category, index) => {
+      return (
+        <option key={index} value={category}>
+          {category}
+        </option>
+      );
+    });
+
+    if (title === 'category') {
+      return (
+        <select className="restaurant-filter category" onChange={changeEvent}>
+          {option}
+        </select>
+      );
+    }
+
+    if (title === 'sort') {
+      return (
+        <select className="restaurant-filter sort" onChange={changeEvent}>
+          {option}
+        </select>
+      );
+    }
+  }
```

Comment:
조건문으로 처리하신 이유가 있을까요? FilterList 에 만약 날짜순.. 인기순.. 이런것들이 생기면 FilterList 컴포넌트에 if문이 계속 추가되게 되지는 않을까요?

Reply:
각 필터에 맞는 클래스 이름을 설정해주려 했었는데 불필요하게 조건문을 사용했던 것 같습니다. 
아래와 같이 바꿀 수 있을 것 같습니다!

```suggestion
    return (
      <select className={`restaurant-filter ${title}`} onChange={changeEvent}>
        {option}
      </select>
    );
  }
```